README.md

```diff
@@ -283,6 +260,6 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This Link is broken because you did not add a license to your project. please do by looking at [this resource](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 😄